app/src/main/java/de/westnordost/streetcomplete/user/QuestStatisticsByQuestTypeFragment.kt

```diff
@@ -51,6 +51,7 @@ class QuestStatisticsByQuestTypeFragment : Fragment(R.layout.fragment_quest_stat
         })
     }
 
+    @SuppressLint("UseCompatLoadingForDrawables")
```

Comment:
Why?

Reply:
A warning that `ResourcesCompat.getDrawable()` should be used is displayed.

Reply:
And why silence the warning?

Reply:
Or asked another way: Why is there a warning for using `ResourcesCompat.getDrawable()`? There is also a warning for using `Resources.getDrawable`, why not silence this instead, or just leave it as it is?